src/test/java/nl/tudelft/contextproject/model/entities/EntityTest.java

```diff
@@ -1,123 +1,126 @@
-package nl.tudelft.contextproject.model;
-
-import static org.junit.Assert.*;
-import static org.mockito.Mockito.*;
-
-import org.junit.Test;
-
-import com.jme3.math.Vector3f;
-import com.jme3.scene.Spatial;
-
-import nl.tudelft.contextproject.Main;
+package nl.tudelft.contextproject.model.entities;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import nl.tudelft.contextproject.model.Drawable;
+import org.junit.Test;
+
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Spatial;
+
+import nl.tudelft.contextproject.Main;
 import nl.tudelft.contextproject.test.TestUtil;
-
-/**
- * Abstract test class for Entity.
- */
-public abstract class EntityTest extends DrawableTest {
-	
-	private Entity entity;
-	
-	@Override
-	public Drawable getDrawable() {
-		return getEntity();
-	}
-	
-	/**
-	 * Getter for a specific instance of Entity.
-	 * @return an Entity to test with.
-	 */
-	public abstract Entity getEntity();
-
-	/**
-	 * create a new (clean) entity to test with.
-	 */
-	public void setupEntity() {

```

Comment:
Adding `@Before` here would make a lot of tests cleaner.

Perhaps this should be done in a separate PR. If people agree, I'm fine with creating that PR.